### ISSUE TITLE
'Settings has stopped working' on Touchwiz roms

### ISSUE THREAD

**Issue by m0m4x** [m0m4x/AdvDataPlan#1]
[2017-07-14-11-11-32.txt](https://github.com/m0m4x/AdvDataPlan/files/1167311/2017-07-14-11-11-32.txt)
```
07-14 11:11:19.005 E/Xposed  (2292): java.lang.ClassCastException: android.widget.ScrollView cannot be cast to android.widget.LinearLayout
07-14 11:11:19.005 E/Xposed  (2292): 	at net.m0m4x.android.xposed.advdataplan.HookMain$1.handleLayoutInflated(HookMain.java:139)
```

```
07-14 11:11:19.005 E/AndroidRuntime(2292): FATAL EXCEPTION: main
07-14 11:11:19.005 E/AndroidRuntime(2292): Process: com.android.settings, PID: 2292
07-14 11:11:19.005 E/AndroidRuntime(2292): java.lang.NullPointerException: Attempt to invoke virtual method 'void android.widget.NumberPicker.setDisplayedValues(java.lang.String[])' on a null object reference
07-14 11:11:19.005 E/AndroidRuntime(2292): 	at net.m0m4x.android.xposed.advdataplan.HookMain$4.replaceHookedMethod(HookMain.java:370)
```

**Comment by m0m4x** [comment]
Fixed with v1.0.4

**Comment by m0m4x** [comment]
Fixed with v1.0.4
